### PR TITLE
ironPort modeling rules

### PR DESCRIPTION
**Pull request by guytamir10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/IronPort/README.md [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
-- Reporting - get Cisco SMA's statistics reports.
-- Fetch quarantine messages as incidents.
-
-This pack contains an integration, whose main purpose is to detect and block a wide variety of email-borne threats, such as malware, spam and phishing attempts.
+# Cisco ESA (Email Security Appliance)
+
+# Integration:
+The Cisco Email Security Appliance is an email security gateway product. It is designed to detect and block a wide variety of email-born threats, such as malware, spam and phishing attempts.
+
+## What does this pack do?
```
Comment (ShirleyDenkberg) [inline Packs/IronPort/README.md:6]: Not sure this belongs in a Modeling Rule pack readme.  Please check.

File: Packs/IronPort/README.md [hunk 2]
```diff
@@ -1,13 +1,42 @@
...
+- Release and delete messages from spam quarantine.
+- Retrieve, add, append, edit or delete a list entry - blocklist and safelist of spam quarantine. 
+- Retrieve tracking messages.
+- Retrieve tracking messages enrichment summaries - AMP, DLP, URL.
+- Reporting - get Cisco SMA's statistics reports.
+- Fetch quarantine messages as incidents.
+
+## Creating a Log Subscription in the GUI
+
+1. Choose System Administration > Log Subscriptions.
```
Comment (ShirleyDenkberg) [inline Packs/IronPort/README.md:17]:
```suggestion
1. Click **System Administration** > **Log Subscriptions**.
```

File: Packs/IronPort/README.md [hunk 3]
```diff
@@ -1,13 +1,42 @@
...
+2. Click Add Log Subscription.
```
Comment (ShirleyDenkberg) [inline Packs/IronPort/README.md:18]:
```suggestion
2. Click **Add Log Subscription**.
```

File: Packs/IronPort/README.md [hunk 4]
```diff
@@ -1,13 +1,42 @@
...
+3. Select a log type and enter the log name (for the log directory) as well as the name for the log file itself.
+4. Specify the maximum file size before AsyncOS rolls over the log file as well as a time interval between
+rollovers. See Rolling Over Log Subscriptions, page 39-48 for more information on rolling over log files.
```
Comment (ShirleyDenkberg) [inline Packs/IronPort/README.md:21]: Where is the information?  Need a link

File: Packs/IronPort/ReleaseNotes/2_0_9.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+
+#### Modeling Rules
+
+##### New: Cisco ESA Modeling Rule
+
+- Created a modeling rules.
```
Comment (ShirleyDenkberg) [inline Packs/IronPort/ReleaseNotes/2_0_9.md:6]:
```suggestion
Created Cisco ESA modeling rules.
```

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

File: Packs/IronPort/README.md (hunk 4 above)
Comment (guytamir10) [reply Packs/IronPort/README.md:21]: attached link bellow

**Pull request merged**